src/V3TSP.h

```diff
@@ -49,7 +49,7 @@ using StateVec = std::vector<const TspStateBase*>;
 
 // Given an unsorted set of TspState's, sort them to minimize
 // the transition cost for walking the sorted list.
-void tspSort(const StateVec& states, StateVec* resultp) VL_MT_DISABLED;
+void tspSort(const StateVec& states, StateVec* resultp) VL_MT_STABLE;
```

Comment:
Please add VL_PURE to definition too.  Shouldn't the checker be warning about that?

More critically that function adds edges and other stuff so I don't see it as stable.

Reply:
Marked as `VL_MT_SAFE` since it uses global dumping functions (such as `dumpGraphLevel`). Definition was also marked.

Reply:
I think you mean marked STABLE.  Surprising but seems true.